dspace-uclouvain/src/main/java/org/dspace/uclouvain/pdfAttestationGenerator/factory/PDFAttestationGeneratorFactory.java

```diff
@@ -0,0 +1,16 @@
+package org.dspace.uclouvain.pdfAttestationGenerator.factory;
+
+import java.util.UUID;
+
+import org.dspace.services.factory.DSpaceServicesFactory;
+import org.dspace.uclouvain.pdfAttestationGenerator.handlers.PDFAttestationGeneratorHandler;
+
+public interface PDFAttestationGeneratorFactory {
+    public PDFAttestationGeneratorHandler getHandlerInstance(UUID uuid);
+
+    static PDFAttestationGeneratorFactory getInstance() {
+        return DSpaceServicesFactory.getInstance().getServiceManager().getServiceByName(
+            "pdfAttestationGeneratorFactory", PDFAttestationGeneratorFactory.class
+        );
```

Comment:
COSMETIC: 
I think this is more readable 
```java
return DSpaceServicesFactory
    .getInstance()
    .getServiceManager()
    .getServiceByName("pdfAttestationGeneratorFactory", PDFAttestationGeneratorFactory.class);
```

this is very personal feeling, use it if you think it could be better